fixes to README

konstagap opened the pull request.

alfjjacob commented on .env.example:
```diff
@@ -1,24 +1,23 @@
 #Azure Service for server side validation
-AZURE_TENANT_ID=
-AZURE_CLIENT_ID=
+AZURE_TENANT_ID= [ Azure Tenant ID used by passport-azure-ad package for tokens validation ]
```
Maybe, add - this is required only if ADFS is the Authentication provider

alfjjacob commented on .env.example:
```diff
@@ -1,24 +1,23 @@
-AUTH_CLIENT_ID=
+AUTH_URL= [HPCC Auth Service url]
+AUTH_PORT= [HPCC Auth Service port]
+AUTH_CLIENT_ID= [HPCC Auth Service client id]
```
A description for these fields will be useful. Also, it may be good to group all Auth Service and ADFS related configs in different sections

AUTH_CLIENT_ID=#required only if AuthService is the Authentication provider. This is a unique identifier for an application in Auth Service. Will be setup through Auth Service when a new Application is registered

alfjjacob commented on .env.example:
```diff
@@ -34,7 +33,7 @@ DB_USER=
 DB_ROOT_PASS=
 DB_NAME=real_bi
-DB_HOST=db
+DB_HOST= [if you running app localy use 'localhost' if u using docker 'db' ]
```
typo

alfjjacob commented on README.md:
```diff
@@ -8,54 +8,46 @@ Real BI is a tool used to connect to [HPCC](https://hpccsystems.com/) and create
-2. Rename the **_.env.example_** file to **_.env_** and fill in the remaining values.
-3. Run `npm install` in root folder
+1. Run `git clone https://github.com/hpcc-systems/REAL-BI.git` to copy project into your computer.
+2. Rename the **_.env.example_** file to **_.env_** and fill in the empty values. You can find explenation inside **_.env.example_** .
```
explanation

alfjjacob commented on README.md:
```diff
@@ -8,54 +8,46 @@ Real BI is a tool used to connect to [HPCC](https://hpccsystems.com/) and create
+
+5. Create `real_bi` schema in your local MySQL database.
+6. Go to `/api` and run `npx sequelize db:migrate` to build database tables.
+7. At least one cluster will need to be added to the database to begin making dashboards. A sample seed file is provided in **_api/src/seeders_**.
```
Maybe, mention 'HPCC Cluster'

alfjjacob commented on README.md:
```diff
@@ -8,54 +8,46 @@ Real BI is a tool used to connect to [HPCC](https://hpccsystems.com/) and create
+   - Change the `NODE_ENV` value to "production".
+   - Change the database information to point to your production database.
+3. Rename the **_.env.example_** file to **_.env_** and fill in the empty values.You can find explenation inside **_.env.example_** .
+4. Add cert and key files to `/nginx/certs`.
```
Should we also mention about the steps to be followed incase someone wants to run the app without SSL in docker? i.e. Comment out the ssl related configs in nginx

konstagap commented:
Hi @alfjjacob. I have fixed the issues that you have mentioned, also I have added an image from demo dashboards, and a script to run dependency installation with one command from the root folder.
